src/pytom_tm/tmjob.py

```diff
@@ -16,13 +16,16 @@
 from pytom_tm.io import read_mrc_meta_data, read_mrc, write_mrc, UnequalSpacingError
 
 
-def load_json_to_tmjob(file_name: pathlib.Path) -> TMJob:
+def load_json_to_tmjob(file_name: pathlib.Path, load_for_extraction: bool = True) -> TMJob:
```

Comment:
With this and the previous PR, we should also probably update the version number on our pyproject.toml

Reply:
Good point, will update !